Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,72 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0px;
+    padding: 0px;
+}
+body{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height:100vh ;
+}
+.square-container{
+    background-color: aqua;
+    height: 400px;
+    width: 400px;
+    position: relative;
+    
+}
+.square{
+    background-color: black;
+    height: 100px;
+    width: 100px;
+    position: absolute;
+    
+
+}
+.top{
+    margin-top: -40px;
+    top:0;
+    left: 150px;
+}
+.buttom{
+    margin-bottom: -40px;
+    bottom:0;
+    left: 150px;
+   
+}
+.right{
+    margin-right: -40px;
+    top:150px;
+    right:0 ;
+}
+.left{
+    margin-left: -40px;
+    top: 150px;
+    left: 0;
+}
+.center{
+    height: 200px;
+    width: 200px;
+    background-color: red;
+    position: absolute;
+    border-radius: 100px;
+    top: 100px;
+    left: 100px;
+   
+   
+}
+.center2{
+    height: 100px;
+    width: 100px;

```

Comment:
The background-color value should be 'rgb(19, 17, 17)' instead of 'rgb(17, 17, 17)' for the 'center2' class.